### ISSUE TITLE
Automate QA issue creation 

### ISSUE THREAD

**Issue by Draeieg** [ubiquibot/QA-Improvements#1]
Currently before starting a QA I have to 

1. Go [here](https://github.com/ubiquity/ubiquibot/issues?q=is%3Aissue+is%3Aclosed)
2. Identify closed issued with QA pending (It might seems impossible but I've missed some)
3. Open a QA issue [here ](https://github.com/ubiquibot/staging/issues) using our template manually
4. Add the original closed issue to our QA [project](https://github.com/orgs/ubiquity/projects/23/views/1) 

Automating this would allow me or anyone in doing QA to instantly jump to test and notice quickly when something new is pending 

## Possible solution
We can use GitHub projects as an array solving all of these problems

### Create a custon YML action separated from the bot 

this action should:

 - Add every issue with a PR that needs my review to the QA project (maybe this step can be done using a custom workflow but I'm not sure
 - create the QA issue 
 - Once the QA issue is closed as completed the original issue is marked as passed in the QA project

**Comment by pavlovcik** [comment]
Shouldn't setting the assignee have been automatic upon opening the pull request?

**Comment by 0xcodercrane** [comment]
TL;DR about the QA of this issue.
There is an issue that the workflow file doesn't read the secrets if the PR is created from the branch of the forked repo to the `development` branch. 

1/ [Internal PR](https://github.com/ubiquity/ubiquibot/pull/455), [Workflow Run](https://github.com/ubiquity/ubiquibot/actions/runs/5451826153/jobs/9918532126)

2/ [External PR](https://github.com/ubiquity/ubiquibot/pull/454), [Workflow Run](https://github.com/ubiquity/ubiquibot/actions/runs/5451853513/jobs/9918592861)

The linked QA only contains the case that the PR is created between internal branches.

@wannacfuture. Please take a look and let me know if you have any more questions. hope you open a PR asap.
